src/app/containers/LiquidityMining/components/RewardPool.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { backendUrl, currentChainId } from '../../../../utils/classifiers';
+import axios from 'axios';
+
+import { translations } from 'locales/i18n';
+
+import { RewardTable, WeekRewardType } from './RewardTable';
+
+export interface Props {
+  isConnected: boolean;
+  txList: Array<any>;
+  user: string;
+}
+
+export function RewardPool(props: Props) {
+  const { t } = useTranslation();
+  const api = backendUrl[currentChainId];
+  const [data, setData] = useState<WeekRewardType[]>([]);
+
+  useEffect(() => {
+    if (props.user) {
+      axios
+        .get(api + 'amm/liquidity-mining/sov-calc/' + props.user)
+        .then(res => {
+          const { week1, week2, week3, week4 } = res.data;
+          const now = new Date();
+          const weeks: WeekRewardType[] = [];
+          if (now.getTime() - new Date(week1.weekStart).getTime() > 0) {
+            weeks.push({ ...week1, rewardPool: 30000 });
+          }
+          if (now.getTime() - new Date(week2.weekStart).getTime() > 0) {
+            weeks.push({ ...week2, rewardPool: 15000 });
+          }
+          if (now.getTime() - new Date(week3.weekStart).getTime() > 0) {
+            weeks.push({ ...week3, rewardPool: 15000 });
+          }
+          if (now.getTime() - new Date(week4.weekStart).getTime() > 0) {
+            weeks.push({ ...week4, rewardPool: 15000 });
+          }
+          setData(weeks);
+        })
+        .catch(e => console.error(e));
+    }
+  }, [api, props.user]);
```

Comment:
- Repeat request every 60 seconds
- Make previous request is canceled when another request starts or component is unmounted